File: src/main/java/com/optica/domain/Apply.java
package com.optica.domain;

import java.time.LocalDate;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Entity
public class Apply {

	@Id
	@GeneratedValue
	private Long id;

	private Long curriculumId;

	private Long jobOfferId;

	private LocalDate applyDate;

	private Boolean active;

	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public Long getCurriculumId() {
		return curriculumId;
	}

	public void setCurriculumId(Long curriculumId) {
		this.curriculumId = curriculumId;
	}

	public Long getJobOfferId() {
		return jobOfferId;
	}

	public void setJobOfferId(Long jobOfferId) {
		this.jobOfferId = jobOfferId;
	}

	public LocalDate getApplyDate() {
		return applyDate;
	}

	public void setApplyDate(LocalDate applyDate) {
		this.applyDate = applyDate;
	}

	public Boolean getActive() {
		return active;
	}

	public void setActive(Boolean active) {
		this.active = active;
	}

}

File: src/main/kotlin/com/optica/domain/Apply.kt
package com.optica.domain

import java.time.LocalDate

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.Id

@Entity
class Apply {

    @Id
    @GeneratedValue
    var id: Long? = null

    var curriculumId: Long? = null

    var jobOfferId: Long? = null

    var applyDate: LocalDate? = null

    var active: Boolean? = null

}
